embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/BuildInfo.kt

```diff
@@ -19,13 +19,19 @@ public class BuildInfo(
     /**
      * The Flavor name of the particular build, extracted at compile-time.
      */
-    public val buildFlavor: String?
+    public val buildFlavor: String?,
+
+    /**
+     * The ID of the particular js bundle, generated at compile-time.
+     */
+    public val bundleId: String?,
 ) {
 
     public companion object {
         public const val BUILD_INFO_BUILD_ID: String = "emb_build_id"
         public const val BUILD_INFO_BUILD_TYPE: String = "emb_build_type"
         public const val BUILD_INFO_BUILD_FLAVOR: String = "emb_build_flavor"
+        public const val BUILD_INFO_BUNDLE_ID: String = "emb_bundle_id"
```

Comment:
We should make clear what this is - if it's the RN JS bundle ID, we should called it `emb_js_bundle_id` or something that makes it clear it is not the Android build bundle's ID. Ideally the swazzler attribute should be changed too, but we should change at least the constant in the Android SDK.